### PR TITLE
Uml 1187 show how many lpas added

### PR DESCRIPTION
**Pull request by GemTay**
# Purpose

Adds an LPA count to the dashboard page

Fixes UML-1187

## Approach

Added LPA count content to the dashboard template, updated translation files. Also changed title of Older LPA page that is incorrect & the same as the Cannot send activation key page title

## Checklist

* [x] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [x] I have added tests to prove my work
* [ ] I have added welsh translation tags and updated translation files
* [ ] I have run an accessibility tool on any pages I have made changes to and fixed any issues found
* [ ] The product team have tested these changes

### CODE REVIEW

File: service-front/app/languages/cy/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -5157,3 +5160,13 @@ msgstr ""
 "Os ydych wedi gwneud <abbr title=\"atwrneiaeth arhosol\">LPA</abbr> ar-lein, "
 "mae angen ichi <a href=\"%link%\" class=\"govuk-link\">greu cyfrif ar wahân</"
 "a> i ddefnyddio’r gwasanaeth hwn yr un fath."
+
+#: src/Actor/templates/actor/lpa-dashboard.html.twig:56
+msgid "You have %count% LPA in your account"
+msgid_plural "You have %count% LPAs in your account"
+msgstr[0] "Mae gennych %count% LPA ar eich cyfrif"
```
Comment (cooperaj) [inline service-front/app/languages/cy/LC_MESSAGES/messages.po:5167]: Just for context this is 
0 : 0 (0)
1 : 1 (1)
2 : 2 (2)
3 : few (3)
4 : many (6)
5 : other (4, 5, 7~20, 100, 1000, 10000, 100000 ...)

**Comment by GemTay** [comment]
image

File: service-front/app/features/context/UI/LpaContext.php [hunk 2]
```diff
@@ -307,6 +307,53 @@ public function iAmToldThatIHaveAnActivationKeyForThisLpaAndWhereToFindIt()
...
+                        StatusCodeInterface::STATUS_OK,
+                        [],
+                        json_encode([])
+                    )
+                );
+
+            // change the token within the LPA data to match as it changes
+            $this->lpaData['user-lpa-actor-token'] = $this->userLpaActorToken;
+            $lpas[$this->userLpaActorToken] = $this->lpaData;
+            $this->userLpaActorToken = (string) (intval($this->userLpaActorToken) + 1);
```
Comment (GemTay) [inline service-front/app/features/context/UI/LpaContext.php:343]: For clarity, this loop and changing of the token is to replicate this format that is returned from the `/v1/lpas` request below
image

File: service-front/app/src/Actor/templates/actor/already-have-activation-key.html.twig [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {% extends '@actor/layout.html.twig' %}
 
-{% block html_title %}{% trans %}Cannot send an activation key{% endtrans %} - {{ parent() }}{% endblock %}
+{% block html_title %}{% trans %}You have an activation key for this LPA{% endtrans %} - {{ parent() }}{% endblock %}
```
Comment (GemTay) [inline service-front/app/src/Actor/templates/actor/already-have-activation-key.html.twig:3]:
```suggestion
{% block html_title %}{% trans %}Cannot send an activation key{% endtrans %} - {{ parent() }}{% endblock %}
```

**Review by cooperaj — DISMISSED** [review]

File: service-front/app/features/actor-dashboard.feature [hunk 4]
```diff
@@ -64,3 +64,15 @@ Feature: The user is able to see correct information on their dashboard
...
     And I click the Back to your LPAs link on the page
     Then I should be taken to the <dashboard> page
+
+  @ui
+  Scenario Outline: As a user I am shown how many LPAs are in my account
+    Given I have added <count> LPAs to my account
+    When I am on the dashboard page
+    Then I am told that I have <count> LPAs in my account
+
+    Examples:
```
[the remark this answers is not in the document]
Comment (GemTay) [reply service-front/app/features/actor-dashboard.feature:74]: True! I seem to remember seeing a ticket in the backlog around a future design where we display LPAs in groups of 10 on separate pages in the dashboard eg Showing 10-20 of 35 LPAs. I guess the functionality of this test could be useful if that design gets implemented?

File: service-front/app/languages/cy/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -2650,11 +2657,6 @@ msgstr ""
 msgid "Why is this?"
 msgstr "Pam?"
 
-#: src/Actor/templates/actor/partials/lpa-details.html.twig:50
```
Comment (cooperaj) [inline service-front/app/languages/cy/LC_MESSAGES/messages.po:2653]: I don't see these being added in anywhere else in the file. Should they have been removed?

File: service-front/app/languages/cy/LC_MESSAGES/messages.po [hunk 6]
```diff
@@ -2699,16 +2706,6 @@ msgstr "WEDI DOD I BEN"
...
-#: src/Actor/templates/actor/partials/lpas-registered-after.html.twig:8
-msgid ""
-"You can only use this service if we've sent you a letter or email with an "
-"activation key. Only LPAs registered after 16 July 2020 have an activation key."
-msgstr ""
-"I ddefnyddio’r gwasanaeth hwn, mae’n rhaid i chi fod wedi derbyn llythyr neu e-"
-"bost yn cynnwys allwedd cadarnhau cyfrif i wneud y cyfrif yn weithredol. Dim "
-"ond <abbr title=\"atwrneiaethau arhosol\">LPAs</abbr> a gofrestrwyd ar ôl yr "
-"16 Gorffennaf 2020 sydd ag allwedd cadarnhau cyfrif."
-
```
Comment (GemTay) [inline service-front/app/languages/cy/LC_MESSAGES/messages.po:2711]: This was removed during extraction as this line of content was changed

File: service-front/app/languages/cy/LC_MESSAGES/messages.po (hunk 5 above)
Comment (GemTay) [reply service-front/app/languages/cy/LC_MESSAGES/messages.po]: I've added this back in, not sure why it had been removed

**Review by cooperaj — APPROVED** [review]

**Pull request merged**